addons/live-comments/script-project-studios.js

```diff
@@ -0,0 +1,158 @@
+const wait = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+
+export default async function ({ addon, global, console, msg }) {
+  let eButton = document.createElement("button");
+  let enabled = false;
+
+  async function addButton() {
+    await addon.tab.waitForElement("h4>span, h2>span", { markAsSeen: true });
+    let commentsText = Array.from(document.querySelectorAll("h4>span, h2>span")).find(
+      (el) => el.innerText == "Comments"
+    );
+
+    function setButtonStyles() {
+      eButton.innerHTML = enabled ? "Live on" : "Live off";
+      eButton.classList.add(enabled ? "live-button-enabled" : "live-button-disabled");
+      eButton.classList.remove(!enabled ? "live-button-enabled" : "live-button-disabled");
+      //   refreshComments()
+    }
+
+    eButton.onclick = (e) => {
+      enabled = !enabled;
+      setButtonStyles();
+    };
+
+    eButton.classList.add("live-button");
+    setButtonStyles();
+
+    // place button
+    commentsText?.after(eButton);
+  }
+
+  (async () => {
+    while (true) {
+      await addButton(); /*await wait(1000)*/
+    }
+  })();
```

Comment:
Would `setInterval` work here?

```suggestion
setInterval(() => {
  addButton();
});
```

Reply:
`addButton` uses waitForElement internally, so I don't see why this needs to be changed.